app/src/main/java/fi/metropolia/movesense/bluetooth/MovesenseConnector.kt

```diff
@@ -0,0 +1,33 @@
+package fi.metropolia.movesense.bluetooth
+
+import android.bluetooth.BluetoothManager
+import android.content.Context
+import com.movesense.mds.Mds;
+import com.movesense.mds.MdsConnectionListener
+import com.movesense.mds.MdsException
+
+class MovesenseConnector(context: Context) : MdsConnectionListener {
```

Comment:
Connector ei varsinaisesti kuulu skanner-tehtävän alle. Siitä pitää tehdä oma task.

Reply:
Tehdään näin.